[2023-11-06T12:06:05.141+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-27T00:00:00+00:00 [queued]>
[2023-11-06T12:06:05.163+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-27T00:00:00+00:00 [queued]>
[2023-11-06T12:06:05.164+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:06:05.323+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-27 00:00:00+00:00
[2023-11-06T12:06:05.330+0000] {standard_task_runner.py:57} INFO - Started process 876 to run task
[2023-11-06T12:06:05.336+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-27T00:00:00+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmp9me9ppmn']
[2023-11-06T12:06:05.343+0000] {standard_task_runner.py:85} INFO - Job 101: Subtask task_a
[2023-11-06T12:06:05.452+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-27T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:06:05.581+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-27T00:00:00+00:00'
[2023-11-06T12:06:05.583+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:06:05.584+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:06:05.596+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:06:05.599+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:06:05.600+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:06:05.601+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:06:05.618+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:06:05.626+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231027T000000, start_date=20231106T120605, end_date=20231106T120605
[2023-11-06T12:06:05.655+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 101 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 876)
[2023-11-06T12:06:05.678+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:06:05.713+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:10.190+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-27T00:00:00+00:00 [queued]>
[2023-11-06T12:19:10.217+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-27T00:00:00+00:00 [queued]>
[2023-11-06T12:19:10.221+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:10.400+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-27 00:00:00+00:00
[2023-11-06T12:19:10.412+0000] {standard_task_runner.py:57} INFO - Started process 2086 to run task
[2023-11-06T12:19:10.430+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-27T00:00:00+00:00', '--job-id', '173', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpt6ylclka']
[2023-11-06T12:19:10.450+0000] {standard_task_runner.py:85} INFO - Job 173: Subtask task_a
[2023-11-06T12:19:10.566+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-27T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:10.748+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-27T00:00:00+00:00'
[2023-11-06T12:19:10.750+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:10.754+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:10.769+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:10.773+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:10.774+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:10.819+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231027T000000, start_date=20231106T121910, end_date=20231106T121910
[2023-11-06T12:19:10.879+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:10.909+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
